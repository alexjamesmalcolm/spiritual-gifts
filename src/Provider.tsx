import { HelmetProvider } from "react-helmet-async";
import { store } from "./store";
import { Provider as ReduxProvider } from "react-redux";

const Provider = ({ children }: { children: JSX.Element }) => {
  return (
    <ReduxProvider store={store}>
      <HelmetProvider>{children}</HelmetProvider>
    </ReduxProvider>
  );
};

export default Provider;
