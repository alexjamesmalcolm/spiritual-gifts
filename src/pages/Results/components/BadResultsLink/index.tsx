import React from "react";
import { Link } from "react-router-dom";
import styles from "./BadResultsLink.module.css";

const BadResultsLink = () => (
  <div className={styles.container}>
    <p>No results found at this link</p>
    <Link to="/">Return Home</Link>
  </div>
);

export default BadResultsLink;
