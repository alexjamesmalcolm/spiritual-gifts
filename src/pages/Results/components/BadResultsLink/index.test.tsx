import { render } from "@testing-library/react";
import Provider from "Provider";
import BadResultsLink from ".";

describe("BadResultsLink", () => {
  it("should render bad results link page", () => {
    const { container } = render(
      <Provider>
        <BadResultsLink />
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
