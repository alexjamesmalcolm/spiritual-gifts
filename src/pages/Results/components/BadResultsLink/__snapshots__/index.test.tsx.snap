// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BadResultsLink should render bad results link page 1`] = `
<div>
  <div
    class="container"
  >
    <p>
      No results found at this link
    </p>
    <a
      href="/"
    >
      Return Home
    </a>
  </div>
</div>
`;
